frontend/src/components/Dropdown/components/MultiDropdownItem/MultiDropdownItem.tsx

```diff
@@ -57,18 +57,32 @@ export const MultiDropdownItem = ({
         <Flex flexDir="column" minW={0}>
           <Stack direction="row" spacing="0.5rem" align="center">
             {icon ? <Icon as={icon} sx={styles.icon} /> : null}
-            <DropdownItemTextHighlighter
-              inputValue={inputValue}
-              showHoverBg={isHighlighted}
-              textToHighlight={label}
-            />
+            <Text
+              textStyle="body-1"
+              minWidth={0}
+              textOverflow="ellipsis"
+              whiteSpace="nowrap"
+              overflowX="hidden"
+            >
```

Comment:
just to clarify - this is not for sort and highlighting right? this is just to restrict the content to 1 line